### ISSUE TITLE
Accessibility - back button is unlabelled

### ISSUE THREAD

**Issue by D0vydas** [wix/react-native-navigation#8027]
### What happened?

Navigation back button does not have accessibility label

### Was it tested on latest react-native-navigation?

- [x] I have tested this issue on the latest react-native-navigation release and it still reproduces.

### In what environment did this happen?

React Native Navigation version: 
React Native version: 
Has Fabric (React Native's new rendering system) enabled: (yes/no) 
Node version:
